reflectable/lib/capability.dart

```diff
@@ -502,13 +502,13 @@ class _StringInvocation extends StringInvocation {
   final Map<Symbol, dynamic> namedArguments;
 
   @override
-  bool isMethod;
+  late bool isMethod;
 
   @override
-  bool isGetter;
+  late bool isGetter;
```

Comment:
Not sure where we initialize these. Should they not be getters that inspect `kind`?

Reply:
Definitely yes, thanks!